components/table/index.css

```diff
@@ -124,18 +133,24 @@
 	--mod-button-background-color-hover: transparent;
 	--mod-button-background-color-focus: transparent;
 	--mod-button-background-color-down: transparent;
-	--mod-button-content-color-default: var(--spectrum-table-header-text-color);
-	--mod-button-content-color-hover: var(--spectrum-table-header-text-color);
-	--mod-button-content-color-focus: var(--spectrum-table-header-text-color);
-	--mod-button-content-color-down: var(--spectrum-table-header-text-color);
+	--mod-button-content-color-default: var(--mod-table-header-text-color, var(--spectrum-table-header-text-color));
+	--mod-button-content-color-hover: var(--mod-table-header-text-color, var(--spectrum-table-header-text-color));
+	--mod-button-content-color-focus: var(--mod-table-header-text-color, var(--spectrum-table-header-text-color));
+	--mod-button-content-color-down: var(--mod-table-header-text-color, var(--spectrum-table-header-text-color));
```

Comment:
@rise-erpelding So far, I only updated these to have the table mods included the the button passthrough definitions. Do you think I should do that for any of the other mods?